### PR TITLE
Added fixed dae models compatible with OpenWebGL

### PR DESCRIPTION
**Pull request by RDaneelOlivav**
Here we added new versions of the Dae files that are compatible with OpenWebGL and minor material tweeks like the red emergency button.

### CODE REVIEW

**Comment by erelson** [comment]
Could you provide some summary information on what changes are needed to ensure openwebgl compatibility? I would like to pass that info on to the person who historically creates these files.

Additionally, assuming you intend this to be merged eventually, go ahead and give the files the actual names (no .fix) 👍

Thanks!

**Comment by RDaneelOlivav** [comment]
@erelson Just committed the name changes to their original names. Please let me know if you fetchers accepts and merge the changes to the main repo so that I can update everywhere where we are using these files to use the main repo.

For future changes, I suppose this fork and pull request method is the one to use?


As for the method to make dae files compatible with OpenWebGL, its quite tricky and I can't tell you exactly what has to be done with precision. The problem is that WebGL has some issues with certain ways of exporting textures and materials and depending on the program used and method to export those files, they just don't appear or they appear without textures. I recommend using plain textures mapped with UV or materials using Blender render ( not cycles material systems ). Sorry, I can't give you more details, but that's the way we generated these dae files, iteratively until they worked in both Gazebo and WebGL. If someone knows more about the topic please let us know.

File: fetch_description/robots/fetch.urdf [hunk 1]
```diff
@@ -565,7 +565,7 @@
     <child link="laser_link" />
     <axis xyz="0 0 0" />
   </joint>
-  <link name="torso_fixed_link">
+  <link name="torsoed_link">
```
Comment (erelson) [inline fetch_description/robots/fetch.urdf]: Let's keep the original link/joint name of torso_fixed_joint, as well.

Comment (RDaneelOlivav) [reply fetch_description/robots/fetch.urdf]: Ok Fixed I think now. Made a replace to fast ;). 
Tell me if there is anything else and notify me if we merge

**Comment by erelson** [comment]
Thanks for the quick fixes! I've gotta check/test with a few others before going ahead and merging this.

If you're able to provide screenshots in RViz before/after, that will help demonstrate that the changes are tested and look OK.

**Comment by RDaneelOlivav** [comment]
screenshot from 2019-02-19 20-56-31
screenshot from 2019-02-19 21-02-42
screenshot from 2019-02-19 21-03-21

Is that ok? I dont have the footage that showed how it appered before, didnt print it.

**Comment by mfbailey91** [comment]
LGTM

File: fetch_description/robots/fetch.urdf [hunk 2]
```diff
@@ -587,7 +587,7 @@
       </geometry>
     </collision>
   </link>
-  <joint name="torso_fixed_joint" type="fixed">
+  <joint name="torsoed_joint" type="fixed">
```
Comment (erelson) [inline fetch_description/robots/fetch.urdf]: @RDaneelOlivav sorry I didn't specify before, but there were multiple instances needing to be reverted to the original.

Comment (RDaneelOlivav) [reply fetch_description/robots/fetch.urdf]: I thought I had fixed that in this commit: 47b97450cb26776d44131fb847ca52a42d093c29

Is there something I'm missing here? There were two places where there was an error. Is there anywhere else?

Comment (erelson) [reply fetch_description/robots/fetch.urdf]: Yeah, this is weird. Your commit's (the most recent one) diffs look correct, but when viewing the full diffs, the current version of the file is wrong for this line. I can easily fix that and am doing so now.

Comment (erelson) [reply fetch_description/robots/fetch.urdf]: There were actually three places, I see now. :+1:

**Review by erelson — APPROVED** [review]

**Pull request merged**